go/signature/signature_key_templates_test.go

```diff
@@ -19,65 +19,170 @@ import (
 	"testing"
 
 	"github.com/golang/protobuf/proto"
-	"github.com/google/tink/go/signature"
-	"github.com/google/tink/go/testutil"
 	commonpb "github.com/google/tink/go/proto/common_go_proto"
 	ecdsapb "github.com/google/tink/go/proto/ecdsa_go_proto"
 	tinkpb "github.com/google/tink/go/proto/tink_go_proto"
+	"github.com/google/tink/go/signature"
```

Comment:
I guess these should go back before  the proto imports? I will revert the import changes..

Reply:
revert the order of these imports as well